sklearn/cluster/_kmeans.py

```diff
@@ -1576,20 +1576,23 @@ def __init__(self, n_clusters=8, *, init='k-means++', max_iter=100,
         self.init_size = init_size
         self.reassignment_ratio = reassignment_ratio
 
-    @deprecated("The attribute 'counts_' is deprecated in 0.24"  # type: ignore
-                " and will be removed in 0.26.")
+    @deprecated(  # type: ignore
+        "The attribute 'counts_' is deprecated in 0.24"
```

Comment:
We will get conflict with this: https://github.com/scikit-learn/scikit-learn/pull/19005
Do you think that we could merge #19005 first :)

Reply:
Yes, lets merged that first.

Reply:
#19005 is merged